### PR TITLE
Add Anna Persona for Sign In

### PR DESCRIPTION
**Pull request by JamieCleare2525**
## Context

Adding the "Anne" persona for User login without DfE Sign In

## Running

- run `bundle exec rake db:seed` to generate the Persona/User for Anne

## Changes proposed in this pull request

- Add `User` and `Persona` models
- Migration to add `users` to the database
- Create `personas` index page
- Create `account` page

## Link to Trello card

https://trello.com/c/yZSxLwtM/17-set-up-persona-based-login

## Things to check

- [ ] If the code removes any existing feature flags, a data migration has also been added to delete the entry from the database
- [ ] This code does not rely on migrations in the same Pull Request
- [x] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [ ] If this code adds a column to the DB, decide whether it needs to be in analytics yml file or analytics blocklist
- [ ] API release notes have been updated if necessary
- [ ] If it adds a significant user-facing change, is it documented in the [CHANGELOG](CHANGELOG.md)?
- [ ] Required environment variables have been updated or [added to the Azure KeyVault](/docs/environment-variables.md#deploy-pipeline)

## Screenshots
screencapture-localhost-3000-personas-2023-12-11-16_34_30
screencapture-localhost-3000-account-2023-12-11-16_34_07

https://github.com/DFE-Digital/itt-mentor-services/assets/17162488/a67d9294-ebfa-4c9f-bf0e-2ede9d3d2573

### CODE REVIEW

File: app/views/personas/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+            <li><%= role %></li>
+          <% end %>
+        </ul>
+      <% end %>
+
+      <%= form_tag("/auth/developer/callback", method: "post", data: {turbo: false}) do %>
+        <%= hidden_field_tag "email", persona.email %>
+        <%= hidden_field_tag "first_name", persona.first_name %>
+        <%= hidden_field_tag "last_name", persona.last_name %>
+        <button type='submit' class="govuk-button govuk-!-margin-bottom-2">
```
Comment (CatalinVoineag) [inline app/views/personas/index.html.erb:31]: This needs to be double quoted I think

File: app/views/account/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,34 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l">Your account</h1>
+
+    <dl class="govuk-summary-list">
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">
+          First name
```
Comment (CatalinVoineag) [inline app/views/account/index.html.erb:8]: Should we use translations for these?

**Review by CatalinVoineag — COMMENTED** [review]
This looks very good, well done! I've just added minor comments and I have one question.

Should our default route be `/personas`? This is more of a question for the whole team

Also, did you run `bin/lint`? 😆

File: app/views/account/index.html.erb (hunk 2 above)
Comment (JamieCleare2525) [reply app/views/account/index.html.erb]: Resolved in [9f8f3b0](https://github.com/DFE-Digital/itt-mentor-services/pull/12/commits/9f8f3b07860c747a02a99e58c3b84c360cae9354)

File: app/views/personas/index.html.erb (hunk 1 above)
Comment (JamieCleare2525) [reply app/views/personas/index.html.erb]: Resolved in [9f8f3b0](https://github.com/DFE-Digital/itt-mentor-services/pull/12/commits/9f8f3b07860c747a02a99e58c3b84c360cae9354)

**Review by CatalinVoineag — APPROVED** [review]
Dev tested this on local, works as expected 👍🏻

**Review by elceebee — APPROVED** [review]
Nice work! I've tested locally and it all works as expected.

**Review by ollietreend — APPROVED** [review]
Looks awesome ✨ 🚀

**Pull request merged**